cetmix_tower_aws/readme/USAGE.md

```
@@ -0,0 +1,120 @@
+
+> **Disclaimer**: The following example demonstrates one of many possible commands you can create and run with this module. The boto3 library provides access to the full range of AWS services and methods - this is just a starting point to help you get familiar with the integration.
+
+**Example of Cetmix Tower Python Command to List EC2 Instances**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading
null

(MD036, no-emphasis-as-heading)